### DISCUSSION TITLE
Should i use root or docs for my static website?

### DISCUSSION THREAD

**Discussion by thedynomo** [community/community#142686] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm working on someone's site and I want it to look a bit professional so I want to follow best practices. Do I use docs or root? And when should I use either?

**Comment by justdeen** [comment]
Use docs when all the files are uploaded to your repo without using any folders. Use root when you make use of folders.

**Comment by DevKyLeOfficial** [comment]
both methods are valid for hosting a GitHub Pages site, using the docs directory is often recommended due to its organizational benefits and ease of management compared to serving directly from the root directory.

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/142683#discussioncomment-11078623) for reference to why this discussion was closed and locked. Thanks!
